clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1637,31 +1637,42 @@ specifies availability for the current target platform, the availability
 attributes are ignored. Supported platforms are:
 
 ``ios``
+``ios_app_extension``
```

Comment:
The description only describes the platform. Can you also include an explanation for app_extension?

Reply:
I can. These paragraphs are already super wordy though, especially compared to the platform descriptions at https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#available. Maybe it would be better to list all the values together and describe them all in a single paragraph below?

Reply:
I actually feel like these don't need descriptions and we don't need to explain how client code sets its deployment target. I think less is more and let's just list the allowed platforms?

Reply:
Cyndy made that point that `-target` and -mtargetos` aren't documented anywhere else. Put together a small paragraph describing them generally so that we don't have to have a long description for every single supported platform/variant.

Reply:
I do think it's still useful to define what those platforms' values are, for the most part, it is an easy search away though.